pkg/ocm/sca/sca.go

```diff
@@ -253,10 +287,18 @@ func (c *Controller) requestSCAWithExpBackoff(ctx context.Context, endpoint stri
 			}
 			return true, httpErr
 		}
+
+		err = json.Unmarshal(data, &responses)
+		if err != nil {
+			klog.Errorf("Unable to decode response: %v", err)
+			return true, err
+		}
```

Comment:
I think it would better to move this out of the exponential backoff function call